modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -640,3 +643,193 @@ test.describe('Manage object entries through View Object Entries', () => {
+				{
+					active: false,
+					externalReferenceCode: boundObjectDefinitionERC,
+					label: {
+						en_US: 'BoundObject',
+					},
+					name: 'BoundObject' + getRandomInt(),
+					objectFields: [
+						{
+							DBType: 'String',
+							businessType: 'Text',
+							externalReferenceCode: 'text',
+							indexed: true,
+							indexedAsKeyword: true,
+							label: {
+								en_US: 'BoundText',
+							},
+							name: 'text',
+							required: false,
+							system: false,
+							type: 'String',
+						},
+					],
+					panelCategoryKey: '',
+					pluralLabel: {
+						en_US: 'BoundObject',
+					},
+					rootObjectDefinitionExternalReferenceCode:
+						rootObjectDefinitionERC,
+					scope: 'company',
+					status: {
+						code: 2,
+					},
+					titleObjectFieldName: 'text',
+				}
+			);
+
+		await apiHelpers.objectAdmin.postObjectDefinitionPublish(
+			rootObjectDefinition.id
+		);
+
+		const applicationName =
+			'c/' + rootObjectDefinition.name.toLowerCase() + 's';
+
+		const textObjectEntry = {
+			text: 'Root',
+		};
+
+		const objectEntry = await apiHelpers.objectEntry.postObjectEntry(
+			textObjectEntry,
+			applicationName
+		);
+
+		const companyId = await page.evaluate(() => {
+			return Liferay.ThemeDisplay.getCompanyId();
+		});
+
+		const role = await apiHelpers.headlessAdminUser.postRole({
+			name: 'Bound Updater ' + getRandomString(),
+			rolePermissions: [
+				{
+					actionIds: ['UPDATE', 'VIEW'],
+					primaryKey: companyId,
+					resourceName:
+						'com.liferay.object.model.ObjectDefinition#' +
+						rootObjectDefinition.id,
+					scope: 1,
+				},
+				{
+					actionIds: ['ACCESS_IN_CONTROL_PANEL'],
+					primaryKey: companyId,
+					resourceName:
+						'com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_' +
+						rootObjectDefinition.id,
+					scope: 1,
+				},
+			],
+		});
+
+		const user =
+			await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+				'demo.unprivileged@liferay.com'
+			);
+
+		await apiHelpers.headlessAdminUser.assingUserToRole(role.name, user.id);
+
+		await page.getByLabel('Test Test User Profile').click();
+		await page.getByRole('menuitem', {name: 'Sign Out'}).click();
+
+		await performLogin(page, user.alternateName);
+
+		await viewObjectEntriesPage.goto(rootObjectDefinition.id);
+
+		await page.getByRole('link', {name: '' + objectEntry.id}).click();
+		await page
+			.getByRole('link', {name: boundObjectDefinition.label.en_US})
+			.click();
+
+		await viewObjectEntriesPage.clickAddObjectEntry();
+		await viewObjectEntriesPage.fillObjectEntry({
+			objectFieldName: 'BoundText',
+			objectFieldValue: 'Bound',
+			roleLocator: 'textbox',
+		});
+		await viewObjectEntriesPage.saveObjectEntryButton.click();
+		await waitForSuccessAlert(page);
+
+		await viewObjectEntriesPage.backButton.click();
+
+		await expect(
+			page.getByTestId('visualization-mode-table').getByText('Bound')
+		).toBeVisible();
+
+		// Clean up
```

Comment:
Hey @G-Sa, we can think of other approaches to cleaning the data related to the root model, we can start a discussion about running these tests in another instance. We can also look for other suggestions related to this topic.

Reply:
Hey Sa, can you check the feasibility of creating a new instance?